### ISSUE TITLE
[BUG] : Base option not working when relative

### ISSUE THREAD

**Issue by aecz** [compodoc/compodoc#35]
```
compodoc -p src\tsconfig.json -b .
```
My use case is to be able to navigate the site **without serving it**, just opening the `index.html`.
Setting the base option to `.`  or `""` works but only for the main pages. It fails on module pages for example.

**Comment by vogloblinsky** [comment]
Hi,
Compodoc documentation is not planned to be used directly by opening index.html and having file:// urls in the browsers.
For example, Lunr.js index json file is loaded with ajax, and will failed in your context.

The only workaround for you is to set the base for your local folder like this example : 
```
compodoc -p src\tsconfig.json -b file://localhost/Volumes/Data/xxx/compodoc-demo-todomvc-angular2/documentation/
```

**Comment by aecz** [comment]
Thanks for the workaround. It works except for links that points to `/`.
If we change this to `/index.html` everything works perfectly.
Do you think it can be done ? Would you accept a PR for this ?

The idea behind this is to allow easier integration in Jenkins/Bamboo, Visual Studio Code/Atom...
For example, Istanbul report can easily being added to a build result in a CI server. Compodoc cannot at the moment.

**Comment by vogloblinsky** [comment]
Yep it can be done.
The test case will be :
- build the compodoc ng2 demo in jenkins and export html pages like another reporter
- open the documentation directly in jenkins interface

**Comment by PhilippSoehnlein** [comment]
Same problem here. Our CI-server (TeamCity) runs compodoc and publishes the generated folder as artifact which is then later displayed on the web frontend of the CI server in an `<iframe>` serving an URL like that: `/repository/download/$buildTaskName/$buildId:id/$artifactName!/index.html`. I can't figure out a value for `base` that works.

**Comment by vogloblinsky** [comment]
Fixed in 0.0.39
